src/SWP/Bundle/ContentBundle/Resources/config/serializer/CmfRoutingBundle/Model.Route.yml

```diff
@@ -0,0 +1,21 @@
+Symfony\Cmf\Bundle\RoutingBundle\Model\Route:
```

Comment:
but it extends `Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\Route` and not `Symfony\Cmf\Bundle\RoutingBundle\Model\Route`